template.yaml

```diff
@@ -2,11 +2,25 @@ kind: Template
 apiVersion: v1
 template: spark
 metadata:
-  name: spark
+  name: apache-spark
+  creationTimestamp: null
+  annotations:
+    description: Apache Spark with 3 worker nodes.
+    iconClass: icon-nodejs
+    tags: quickstart,Apache,Spark,apachespark
 labels:
   app: sparkcluster
 objects:
-
+- apiVersion: v1
+  kind: PersistentVolumeClaim
```

Comment:
It was a explicite wish of our development to have a share storage over all nodes. I'm not sure why.